src/test/java/com/maybe/maybe/service/DeskServiceTest.java

```diff
@@ -67,14 +89,52 @@ public void createFromDTOTest() {
         assertEquals(expectedDesk.getName(), actualDesk.getName());
     }
 
+    @Test(expected = BeanCreationException.class)
```

Comment:
As previous pull request i think it is wrong exception usecase and exception name.